### PR TITLE
The Tilening v1

### PR DESCRIPTION
**Pull request by Twaticus**
## About The Pull Request

Ports smooth tiles from TGMC and textured tiles(with color tweaks) from Skyrat, TauCeti, and Eris. I was sent the sprites on discord and told it was from Skyrat, but the PR credits TauCeti AND Eris so im just going to credit all 3 thanks other servers :) Shoutout to @Imaginos16 for all the help you rock.

Details: Resprites the grey, dark, and white tiles. Adds textured version of grey, dark, and white tiles with variations. Adds smooth tiles. New tiles added to reskinning with rotational support. Sprite changes to yellowsiding, plastic, materialfloor, plaque, reacharge_floor, sepia, chapel, and pinkblack to match the new base tiles. Small consistency adjustments on wood_tile and its broken states. Removes 2 unused dirt floor tiles. Reorganized icons\turf\floors.dmi. Sprite adjustments to various icons in icons\obj\tiles.dmi to match the updated sprites. Consistency resprites for glass_wire and tile_e. Small decal tweaks to fit the new base tiles. 

dreamseeker_z7J61czzMG
Normal, Dark, White, Smooth + textured and size variations.

<details>
  <summary>Some areas on Meta</summary>
  
dreamseeker_9ZHxUAgsed
dreamseeker_8Lnf3szXJx
dreamseeker_1Gxk7vfgJL
dreamseeker_XqIajYJ0g8
</details>

<details>
  <summary>Example of texture tile usage. NO mapping changes in this PR, thats for @EOBGames to do ;)</summary>

dreamseeker_08St8YadId
</details>

## Why It's Good For The Game

60 shades of grey yee yee ass old floor sprite days are over. 

## Changelog
:cl: Twaticus, Imaginos16
imageadd: Resprites (almost) all base floor tiles.
imageadd: Ports smooth tiles & textured versions of grey, black, and white tiles.
imageadd: Dance floor item icons are no longer massive.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
Were these unused? 
image

**Comment by Imaginos16** [comment]
It was a blast helping with this PR. Glad to see its finally up!

**Comment by Twaticus** [comment]
> Were these unused?
> image

as far as i know, yes. i checked on VS and the github and couldnt find any information on them

**Comment by LemonInTheDark** [comment]
aight

**Comment by Yenwodyah** [comment]
Huge downgrade from the current sprites. The old tiles have detail to represent the natural wear and tear that floors accumulate over time. These ones are too flat. It makes them look like wall tiles, not floor tiles.

**Comment by Pumpkinoe** [comment]
hot!! hot!! hot!!

**Comment by Sparkezel** [comment]
> Huge downgrade from the current sprites. The old tiles have detail to represent the natural wear and tear that floors accumulate over time. These ones are too flat. It makes them look like wall tiles, not floor tiles.

I want to know your dealer man, current sprites are way too fucking clean for that, these ones only emphasize the problem. Imo there should be some actual worn down tiles, but that's a problem for later ;)

**Comment by Mickyan** [comment]
Metal sidings should probably get a pass since they're based on the old tiles

**Comment by Twaticus** [comment]
image
if you mean these then theyre fine. the tweaks were just to fill in gaps from where the old tile used to have extra 'grout' bits, and the sidings cover the whole edge so there shouldnt be anything missing

**Comment by SmArtKar** [comment]
We are slowly becoming skyrat

**Comment by cacogen** [comment]
Turns out the final boss of unneeded sprite changes was Twaticus themselves. I don't really have any strong feelings about this, though. Instead of arguably ugly shading no one notices it's just flat. You will get sick of looking at that too in time.

File: code/game/objects/items/stacks/tiles/tile_iron.dm [hunk 1]
```diff
@@ -121,13 +197,73 @@
...
+	merge_type = /obj/item/stack/tile/iron/smooth_edge
+	tile_rotate_dirs = list(SOUTH, NORTH, EAST, WEST)
+
+/obj/item/stack/tile/iron/smooth_large1
+	name = "large smooth tile"
+	singular_name = "large smooth floor tile"
+	icon_state = "tile_smooth_large1"
+	turf_type = /turf/open/floor/iron/smooth_large1
+	merge_type = /obj/item/stack/tile/iron/smooth_large1
+	tile_rotate_dirs = list(SOUTH, NORTH, EAST, WEST)
```
Comment (Fikou) [inline code/game/objects/items/stacks/tiles/tile_iron.dm:258]: why 4 dirs if it only has 2 dirs that change the sprite?

File: code/game/objects/items/stacks/tiles/tile_iron.dm [hunk 2]
```diff
@@ -121,13 +197,73 @@
...
+
+/obj/item/stack/tile/iron/smooth_large2
```
Comment (Fikou) [inline code/game/objects/items/stacks/tiles/tile_iron.dm:260]: these paths suck

Comment (Fikou) [inline code/game/objects/items/stacks/tiles/tile_iron.dm:260]: this path is bad

Comment (Twaticus) [reply code/game/objects/items/stacks/tiles/tile_iron.dm:260]: wym?

File: code/game/objects/items/stacks/tiles/tile_iron.dm (hunk 1 above)
Comment (Twaticus) [reply code/game/objects/items/stacks/tiles/tile_iron.dm:258]: cause im idiot

**Comment by Twaticus** [comment]
i saw the plasma, silver, etc tiles and got angry so heres the new ones
image

**Review by Fikou — APPROVED** [review]
blessed
remove yellow siding you wont

**Comment by optimumtact** [comment]
Maybe if you got rid of that yeeyee ass tileset you might get some spriters on ya team, better yet maybe neinhaus will return

**Pull request merged**